_posts/2000-01-03-redundancy-payment.markdown

```diff
@@ -3,6 +3,7 @@
 layout: exemplar
 
 
+exemplar-service-url: "https://notice.redundancy-payments.org.uk/claims/start"
```

Comment:
@jabley @bruntonspall That suggests we don't really need the service link here, I will feed back to content on this.